### PR TITLE
Update catalog.yml with a eLifeSprint 2020 report

### PR DESCRIPTION
**Pull request by lubianat**
Hello, 

We still do not have it as a full preprint, with a DOI and such. Is it an acceptable contribution anyways?

Best,
Tiago

### CODE REVIEW

File: catalog.yml [hunk 1]
```diff
@@ -141,3 +141,5 @@
 - repo_url: https://github.com/habi/zmk-tooth-cohort-method-manuscript
   html_url: https://habi.github.io/zmk-tooth-cohort-method-manuscript/
   preprint_citation: doi:10.1101/2020.10.27.356998
+- repo_url: https://github.com/lubianat/ann_sprint
+  html_url: https://https://lubianat.github.io/ann_sprint/
```
Comment (dhimmel) [inline catalog.yml:145]:
```suggestion
  html_url: https://lubianat.github.io/ann_sprint/
```

**Review by dhimmel — COMMENTED** [review]
> We still do not have it as a full preprint, with a DOI and such. Is it an acceptable contribution anyways?

Definitely! Really cool project BTW.

If you add a line to your metadata.yaml:

```
thumbnail: content/images/logo.png
```

That image will become the thumbnail for the catalog and when the manuscript is shared on social media. More info at https://github.com/manubot/rootstock/blob/master/USAGE.md#thumbnail

**Comment by lubianat** [comment]
Oops, double https, thanks. 
I`ll add that to the metadata.yml (thanks again). 
It is not in the 1000x1000 format, though, I hope it is not a big issue.

**Comment by dhimmel** [comment]
> It is not in the 1000x1000 format, though, I hope it is not a big issue.

yes does not matter. It will work regardless.

**Pull request merged**